components/fctl/cmd/webhooks/list.go

```diff
@@ -49,11 +50,11 @@ func NewListCommand() *cobra.Command {
 						fctl.Map(res.Cursor.Data,
 							func(src formance.WebhooksConfig) []string {
 								return []string{
-									*src.Id,
+									src.Id,
 									src.CreatedAt.Format(time.RFC3339),
-									fctl.StringPointerToString(src.Secret),
-									*src.Endpoint,
-									fctl.BoolPointerToString(src.Active),
+									src.Secret,
+									src.Endpoint,
+									fmt.Sprintf("%v", src.Active),
```

Comment:
This will be print "true" or "false".
fctl use "Yes" and "No" to represent true/false values.
Use fctl.BoolToString() which is dedicated to this purpose.

Reply:
Done